src/steth/scUSDCv2.sol

```diff
@@ -197,7 +201,7 @@ contract scUSDCv2 is BaseV2Vault {
             _multiCall(callData);
         }
 
-        weth.safeTransfer(address(balancerVault), flashLoanAmount);
+        weth.safeTransfer(address(balancerVault), flashLoanAmount + feeAmount);
```

Comment:
I assume you did it for code quality purposes, but it would be less gas expensive to directly do
`weth.safeTransfer(address(balancerVault), _amounts[0] + _feeAmounts[0]);`

since those flashLoanAmount and feeAmount memory variables are used nowhere else.

Reply:
`flashLoanAmount` is used in two places actually, but you have a point for the `feeAmount`. [fixed](https://github.com/lindy-labs/sandclock-contracts/pull/88/commits/b17706f0e8ec3baca2bf9bec0b12907da97c4442)